Sources/MapboxNavigation/MapView.swift

```diff
@@ -198,4 +198,27 @@ extension MapView {
             return VectorSource.isMapboxStreets(identifiers)
         }
     }
+    
+    /**
+     Attempts to localize road labels into the local language and other labels into the given locale.
+     */
+    func localizeLabels(into locale: Locale) {
+        guard !ResourceOptionsManager.hasChinaBaseURL else { return }
+        
+        let style = mapboxMap.style
+        guard let mapboxStreetsSource = streetsSources().first else { return }
+        
+        let streetsSourceTilesetIdentifiers = tileSetIdentifiers(mapboxStreetsSource.id)
+        let roadLabelSourceLayerIdentifier = streetsSourceTilesetIdentifiers.compactMap { VectorSource.roadLabelLayerIdentifiersByTileSetIdentifier[$0]
+        }.first
+        
+        let localizableLayerIdentifiers = style.allLayerIdentifiers.filter { $0.type == .symbol }
```

Comment:
nit: I would apply `.lazy` to this transformation sequence.

Reply:
Good idea, done.